crates/notion-core/src/error/details.rs

```diff
@@ -513,9 +521,30 @@ Please ensure that the correct command is specified.", command),
             ErrorDetails::InvalidHookOutput { command } => write!(f, "Could not read output from hook command: '{}'
 
 Please ensure that the command output is valid UTF-8 text.", command),
+
+            ErrorDetails::InvalidInvocation { action, name, version } => {
+                let error = format!(
+                    "`notion {action} {name} {version}` is not supported.",
+                    action=action,
+                    name=name,
+                    version=version
+                );
+
+                let call_to_action = format!(
+"To install '{name}' version '{version}', please run `notion {action} {formatted}`. \
```

Comment:
I believe `To {action} '{name}'` is meant here, rather than `To install '{name}'`.

Reply:
Good catch; I'll push a fix; thanks!

Reply:
Fixed in 15b9b01!